Project_source/app/src/main/java/android/clicker/school_live_simulator/Player.kt

```diff
@@ -1,6 +1,126 @@
 package android.clicker.school_live_simulator
 
 class Player {
+    class Bag {
+
+    }
```

Comment:
These classes should have only one instance as far as I understand. Why don't we make them objects?

Reply:
I will make them objects. It is a good point